packages/picasso/src/Drawer/Drawer.tsx

```diff
@@ -89,6 +89,7 @@ export const Drawer = (props: Props) => {
           icon={<CloseMinor16 />}
           onClick={handleOnClose}
           className={classes.closeButton}
+          aria-label='drawer-close-button'
```

Comment:
```suggestion
          aria-label='Close drawer'
```
Imagine how an accessibility reader would read the dashes.